sig-and-wg/sig-core/README.md

```diff
@@ -4,7 +4,7 @@ The purpose of this group is to:
 * Publicly represent Kyma
 * Facilitate the collaboration with external contributors
 
-## Responsibilities
+Responsibilities
 
 The SIG Core:
```

Comment:
> Represents the Kyma contributors and is the main contribution contact for Kyma

I feel this needs rephrasing or separation

> Gauges interest, requests the community feedback, and ensures it is addressed

This also probably needs better rephrasing.  Also there's an unnecessary comma before `and`

Reply:
Actually, our [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/style-and-standards.md#punctuation) advise using the serial comma :)

Reply:
Good to know. I HAVE to read those guidelines one day 😄

Reply:
I highly recommend them :)

Reply:
split first bulelt into two